cmd/kubeadm/app/util/runtime/runtime.go

```diff
@@ -56,9 +56,6 @@ func NewContainerRuntime(execer utilsexec.Interface, criSocket string) (Containe
 
 	if criSocket != constants.DefaultDockerCRISocket {
 		toolName = "crictl"
-		// !!! temporary work around crictl warning:
-		// Using "/var/run/crio/crio.sock" as endpoint is deprecated,
-		// please consider using full url format "unix:///var/run/crio/crio.sock"
 		if filepath.IsAbs(criSocket) && goruntime.GOOS != "windows" {
 			criSocket = "unix://" + criSocket
 		}
```

Comment:
i think we can remove this check entirely?
our default sockets on linux now have unix:// and we warn the user if the socket they provided is missing unix://

Reply:
Agree.

And we can remove it now or later when we turn the warning to error. Either is ok to me.

Reply:
+1 to remove it now.